### PR TITLE
Compatibility for Windows Dev Environments

### PR DESCRIPTION
**Pull request by JstnMcBrd**
(Will resolve Issue #34)

On Windows, Prettier was throwing errors because it expected Unix-style line endings. Now VS Code and Prettier should accept the default line endings of whatever operating system the project is on.

On Windows, `npm` scripts would fail because the Windows terminal did not accept Linux commands or file directories. Now VS Code should default to using Git Bash for the terminal, and a note was added to the README for Windows users to set `npm`'s default script shell to Git Bash.

And on a side note, I also added a setting to tell VS Code to auto-format files on save.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,18 @@ DISCORD_TOKEN=YOUR_TOKEN_GOES_HERE
 
 Go to https://discordapi.com/permissions.html#378091424832 and paste in your bot's client ID to get an invite link.
 
+### Important Note for Windows Users
+
+If you're on Windows, `npm` scripts will not working unless you tell `npm` to use Git Bash as its default shell when running commands.
```
Comment (AverageHelper) [inline README.md:78]: Minor grammar nit:

```suggestion
If you're on Windows, `npm` scripts will not work unless you tell `npm` to use Git Bash as its default shell when running commands.
```

File: README.md [hunk 2]
```diff
@@ -72,6 +73,18 @@ DISCORD_TOKEN=YOUR_TOKEN_GOES_HERE
...
+
+Before continuing, run this command:
+
+```sh
+npm config set script-shell "C:\\Program Files\\Git\\bin\\bash.exe"
```
Comment (AverageHelper) [inline README.md:83]: Added `$` for consistency

```suggestion
$ npm config set script-shell "C:\\Program Files\\Git\\bin\\bash.exe"
```

**Review by AverageHelper — CHANGES_REQUESTED** [review]

**Comment by AverageHelper** [comment]
Making a note here in case I forget: Git's default config follows [the official guidance](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf) for line endings on both Unix and Windows platforms:
- On Unix, the default value of `core.autocrlf` is `input`.
- On Windows, the default value of `core.autocrlf` is `true`.

This arrangement, plus the loosening of Prettier's `endOfLine` rule to `auto`, should mean that line endings "just work" regardless of the development platform, while maintaining Unix line endings in the remote.

**Review by AverageHelper — APPROVED** [review]

**Review by gmacgre — APPROVED** [review]
Honestly, seeing this solved will be really nice for me, since I've been manually entering the commands myself. LGTM

**Pull request merged**